omnigibson/transition_rules.py

```diff
@@ -169,23 +172,33 @@ def step(cls):
                 # og.sim.remove_object(removed_obj)
 
         # Then add new objects
-        for added_obj_attr in added_obj_attrs:
-            new_obj = added_obj_attr.obj
-            og.sim.import_object(new_obj)
-            # By default, added_obj_attr is populated with all Nones -- so these will all be pass-through operations
-            # unless pos / orn (or, conversely, bb_pos / bb_orn) is specified
-            if added_obj_attr.pos is not None or added_obj_attr.orn is not None:
-                new_obj.set_position_orientation(position=added_obj_attr.pos, orientation=added_obj_attr.orn)
-            elif isinstance(new_obj, DatasetObject) and \
-                    (added_obj_attr.bb_pos is not None or added_obj_attr.bb_orn is not None):
-                new_obj.set_bbox_center_position_orientation(position=added_obj_attr.bb_pos,
-                                                             orientation=added_obj_attr.bb_orn)
-            else:
-                raise ValueError("Expected at least one of pos, orn, bb_pos, or bb_orn to be specified in ObjectAttrs!")
-            # Additionally record any requested states if specified to be updated during the next transition step
-            if added_obj_attr.states is not None or added_obj_attr.callback is not None:
-                cls._INIT_INFO[new_obj]["states"] = added_obj_attr.states
-                cls._INIT_INFO[new_obj]["callback"] = added_obj_attr.callback
+        if len(added_obj_attrs) > 0:
+            # TODO: Can we avoid this? Currently Rigid contact checking fails if we import objects dynamically
```

Comment:
Hmm is this a new bug with the new issac? Or is this an old bug that has always been there?

Reply:
I will test removing the stop / start requirement

Reply:
unfortunately, seems to be necessary. get segfaults otherwise